services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -3896,6 +3896,13 @@ public boolean isRTTopicDeletionPermittedByAllControllers(String clusterName, St
     for (Map.Entry<String, ControllerClient> controllerClientEntry: controllerClientMap.entrySet()) {
       StoreResponse storeResponse = controllerClientEntry.getValue().getStore(storeName);
       if (storeResponse.isError()) {
+        if (storeResponse.getError().contains("does not exist")) {
```

Comment:
Can we make `does not exist` constant and use it here as well as where this error is sent? 

Also, should we add few retries for get store?